src/Autocomplete/src/Checksum/ChecksumCalculator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Autocomplete\Checksum;
+
+/** @internal */
```

Comment:
```suggestion
/**
 * @author .... ? 
 * 
 * @internal 
 */
```

(same thing on OptionsAware..Interface ?)